pretyping/vnorm.ml

```diff
@@ -196,7 +196,7 @@ and nf_univ_args ~nb_univs mk env sigma stk =
 and nf_evar env sigma evk stk =
   let evi = try Evd.find sigma evk with Not_found -> assert false in
   let hyps = Environ.named_context_of_val (Evd.evar_filtered_hyps evi) in
-  let concl = EConstr.Unsafe.to_constr @@ Evd.evar_concl evi in
```

Comment:
@SkySkimmer doesn't `nativenorm` suffer the same issue?

Reply:
Looks like it does, or at least `Eval native_compute in _ I.` anomalies.

Reply:
Ok, I'll port the patch to `nativenorm` then.

Reply:
Done.